sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -114,6 +114,7 @@ pub fn execute_transaction_to_effects<Mode: ExecutionMode>(
 
     let is_epoch_change = matches!(transaction_kind, TransactionKind::ChangeEpoch(_));
 
+    // todo: if deny_cert I think the intention here is to call create_effects_without_execution instead of execute_transaction_to_effects
```

Comment:
Agreed that these paths are similar/the same (this can/should be a follow-up though).